vars/buildDockerAndPublishImage.groovy

```diff
@@ -137,8 +137,12 @@ def call(String imageName, Map userConfig=[:]) {
           withCredentials([
             usernamePassword(credentialsId: "${finalConfig.gitCredentials}", passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USERNAME')
           ]) {
+            echo "Configuring git user and email"
+            sh 'git config user.name "jenkins"'
```

Comment:
```suggestion
            sh 'git config user.name "${GIT_USERNAME}"'
```
WDYT?

Reply:
Not sure it's appropriate, these tags can be unrelated to any commiter.